.github/workflows/commit-check.yaml

```
@@ -15,102 +16,71 @@ jobs:
-      - name: Extract Branch Name
-        id: branch
-        run: echo "branch_name=${{ github.head_ref }}" >> $GITHUB_ENV
-      - name: Validate Branch Naming Convention
+      - name: Validate branch name
+        env:
+          BRANCH_NAME: ${{ github.head_ref }}
         run: |
-          branch_name="${{ env.branch_name }}"
+          echo "Validating branch name '$BRANCH_NAME'..."
+          pattern='^[a-z]+/[0-9]+$'
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)